docs/more-info/react-native.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "React Native"
+description: "A library to integrate with Metriport on React Native - including Apple Health integrations."
+---
+
+<Info>
+  As a prerequisite [visit the apple health docs](/more-info/apple-health#initial-setup) and take a
+  look at the initial setup for apple healthkit.
```

Comment:
"Before getting started, visit the [Apple Health docs](/more-info/apple-health#initial-setup) and follow the Initial Setup section to configure HealthKit."